osu.Game/Online/Chat/ChannelManager.cs

```diff
@@ -256,8 +256,21 @@ public void PostCommand(string text, Channel target = null)
                     JoinChannel(channel);
                     break;
 
+                case "chat":
```

Comment:
you're really overloading this PR here. please choose one feature to implement and do that first.